lms/templates/instructor/instructor_dashboard_2/add_coupon_modal.html

```diff
@@ -15,7 +16,7 @@
 
     <div id="coupon-content">
       <header>
-        <h2>${_("Add Coupon Code")}</h2>
+        <h3 class="hd hd-3">${_("Add Coupon Code")}</h3>
```

Comment:
This previously was an h2. Should it still be styled that way? I don't know how to view this modal.